src/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 10px;
+  gap: $main-gap;
+  width: ($main-size * 7) + ($main-gap * 6);
+  background-color: #fff;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $main-size;
+    width: $main-size;
+
+    background-color: $day-bg-color;
```

Comment:
The variable $day-bg-color should be defined in the 'variables.scss' file according to the checklist. Ensure that the variable is declared and used properly.